RealmModelGenerator/test.xcdatamodeld/test.xcdatamodel/contents

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```

Comment:
lets remove the text model if we arnt going to use is. sorry i forgots to delete it :'(